draft-ietf-ppm-dap.md

```diff
@@ -706,6 +706,176 @@ last two weeks of the current time) and reject reports that fall outside of the
+                   .----+---.         .----+---.               .----+---.
+                  | uploaded |       | uploaded |             | uploaded |
+                   '----+---'         '----+---'               '----+---'
+                        |                  |                        |
+                        |<-----------------'                        |
+                                                                    |
+                       ...                                          |
+                                                                    |
+                        |<------------------------------------------'
+                        |<------------ aggregation parameter
+                        |               chosen by Collector
+               .--------+----------.
+              |  reports assigned   |
+              | to aggregation jobs |
+               '--------+----------'
+                        |
+                        v
+        .---------------+------------------+---------------- ... --------.
+        |                                  |                             |
+aggregation job 1                   aggregation job 2          aggregation job j
+        |                                  |                             |
+        |                                  '---------------------.       |
+  .-----+-----------------+-------------------+-- ... --.        |       |
+  |                       |                   |         |        |       |
+aggregation            report 1            report 2   report k   |       |
+parameter    .------------+-------------.     |         |        |       |
+  |          |            |             |     |         |        |       |
+  |       Leader     Helper report    public  |         |        |       |
+  |     report share 1  share 1      share 1  |         |        |       |
+  |          |            |             |     |         |        |       |
+  |     .----+----.  .----+----.        |     |         |        |       |
+  |    | decrypted || decrypted |       |     |         |        |       |
+  |     '----+----'  '----+----'        |  .--+---------+---.    |       |
+  |          |            |             | | reports prepared |   |       |
+  |          v            v             |  '--+---------+---'    |       |
+  |    Leader input   Helper input      |     |         |   .----+-------+---.
+  |       share 1      share 1          |     |         |  | aggregation jobs |
+  |          |            |             |     |         |  |       run        |
+  '--------->+----------->|             |     |         |   '----+-------+---'
+             |<-----------+<------------'     |         |        |       |
+             |            |                   |         |        |       |
+         .---+------------+----.              |         |        |       |
+        | input shares prepared |             v         |        |       |
```

Comment:
What's the difference between "reports prepared" and "input shares prepared"?

Reply:
I think I was missing, and what's not clear, is that we're "zooming in" on a part of the pipeline. Perhaps we should split out the zoomed-in part to its own figure?

Reply:
Actually, I'm going to try one more thing that might make this more clear. It'll take a while, though.

Reply:
OK, I reworked the diagram so that we illustrate the preparation of multiple reports, and then the running of multiple aggregation jobs, as stacks of boxes. Hopefully this makes it clear that all reports are handled the same way while being reasonably compact.

I'm trying to optimize for vertical space so that readers don't have to scroll too much when they look at this, but I'm open to using more room in exchange for greater clarity.